<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="MLI-LAP-180" ignored="0" name="HelpdeskRaiseTicketByLocationTest" tests="1" failures="1" timestamp="2022-03-02T19:24:05 IST" time="46.313" errors="0">
  <testcase name="verifyHelpdeskRaiseTicketByLocation" time="33.868" classname="HelixSenseTests.HelpdeskRaiseTicketByLocationTest">
    <failure type="java.lang.AssertionError" message="ERROR!New Ticket is not created and it is failed expected [false] but found [TestChairTicket82618798]">
      <![CDATA[java.lang.AssertionError: ERROR!New Ticket is not created and it is failed expected [false] but found [TestChairTicket82618798]
at HelixSenseTests.HelpdeskRaiseTicketByLocationTest.verifyHelpdeskRaiseTicketByLocation(HelpdeskRaiseTicketByLocationTest.java:21)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyHelpdeskRaiseTicketByLocation -->
</testsuite> <!-- HelpdeskRaiseTicketByLocationTest -->
